x-pack/test/accessibility/apps/index_lifecycle_management.ts

```diff
@@ -48,61 +55,106 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     throw new Error(`Could not find ${policyName} in policy table`);
   };
 
+  const clickPolicyActionsButton = async (policyName: string) => {
+    const actionsCell = await testSubjects.find(`policyTableCell-actions-${policyName}`);
+    const actionsButton = await actionsCell.findByTestSubject('policyActionsContextMenuButton');
+
+    await actionsButton.click();
+  };
+
   describe('Index Lifecycle Management', async () => {
     before(async () => {
-      await esClient.ilm.putLifecycle({ policy: TEST_POLICY_NAME, body: TEST_POLICY_ALL_PHASES });
-      await common.navigateToApp('indexLifecycleManagement');
+      await esClient.ilm.putLifecycle({ policy: POLICY_NAME, body: POLICY_ALL_PHASES });
     });
 
     after(async () => {
       // @ts-expect-error @elastic/elasticsearch DeleteSnapshotLifecycleRequest.policy_id is required
-      await esClient.ilm.deleteLifecycle({ policy: TEST_POLICY_NAME });
+      await esClient.ilm.deleteLifecycle({ policy: POLICY_NAME });
     });
 
-    it('Create Policy Form', async () => {
-      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
-        return testSubjects.isDisplayed('createPolicyButton');
+    beforeEach(async () => {
+      await retry.waitFor('ILM app', async () => {
+        await common.navigateToApp('indexLifecycleManagement');
+        return testSubjects.exists('sectionHeading');
       });
+    });
+
+    describe('Edit policy form', () => {
+      it('Create policy', async () => {
+        const createButtonTestSubject = 'createPolicyButton';
+        await retry.waitFor('ILM create policy button', async () => {
+          return testSubjects.isDisplayed(createButtonTestSubject);
+        });
 
-      // Navigate to create policy page and take snapshot
-      await testSubjects.click('createPolicyButton');
-      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
-        return (await testSubjects.getVisibleText('policyTitle')) === 'Create policy';
+        // Navigate to create policy page and take snapshot
+        await testSubjects.click(createButtonTestSubject);
+        await retry.waitFor('ILM create policy form', async () => {
+          return (await testSubjects.getVisibleText('policyTitle')) === 'Create policy';
+        });
+
+        // Fill out form after enabling all phases and take snapshot.
+        await indexLifecycleManagement.fillNewPolicyForm('testPolicy', true, true, true, true);
```

Comment:
nit: I tend to prefer passing an object and deconstruct it so on the consumer side we now what the flags mean and we don't depend on the order of the parameters (e.g. `fillNewPolicyForm('testPolicy', { coldEnabled: true, frozenEnabled: false })`.